src/gui/properties/speedplotview.cpp

```diff
@@ -202,6 +204,65 @@ quint64 SpeedPlotView::maxYValue()
     return maxYValue;
 }
 
+namespace
+{
+    using Utils::Misc::SizeUnit;
+
+    // table of supposed nice steps for grid marks to get nice looking quarters of scale
+    const double roundingTable[] = {1.2, 1.6, 2, 2.4, 2.8, 3.2, 4, 6, 8};
```

Comment:
How is this table generated?
So this PR makes the Y label to have value that are multiple from this table?

Reply:
Number from this table become 100% label and give "nice looking" 25% (and 50%, 75% consequently too) label(s).